ta/ta.mk

```diff
@@ -57,6 +57,9 @@ aflags$(sm)	:= $(platform-aflags) $($(sm)-platform-aflags)
 # compiled, these flags are not propagated to the TA
 cppflags$(sm)	+= -include $(conf-file)
 cppflags$(sm) += -DTRACE_LEVEL=$(CFG_TEE_TA_LOG_LEVEL)
+ifeq ($(CFG_TA_SANITIZE_UNDEFINED),y)
```

Comment:
Please add `CFG_TA_SANITIZE_UNDEFINED` to `ta-mk-file-export-vars-$(sm)` above.

Reply:
Done